hibernate-reactive-core/src/test/java/org/hibernate/reactive/BaseReactiveTest.java

```diff
@@ -124,17 +126,17 @@ protected Configuration constructConfiguration() {
 			doneTablespace = true;
 		}
 		//Use JAVA_TOOL_OPTIONS='-Dhibernate.show_sql=true'
-		configuration.setProperty( Settings.SHOW_SQL, System.getProperty(Settings.SHOW_SQL, "false") );
+		configuration.setProperty( Settings.SHOW_SQL, System.getProperty(Settings.SHOW_SQL, "true") );
```

Comment:
@gbadner could you roll back this change please?

P.S. A more convenient way to enable SQL logging during development is to set:

    logger.hibernate.level = debug

in `log4j.properties`.

Reply:
@gavinking, this is done.